doc/Language/5to6-perlop.pod6

```diff
@@ -11,15 +11,15 @@ equivalents with notes on variations between them where necessary.
 
 =head1 NOTE
 
-I will I<not> be explaining the operators in detail. This document is an
-attempt to guide you from the operators in Perl 5's perlop document to
+This documentI  I<does not> explains the operators in detail.
+This document is an attempt to guide you from the operators in Perl 5's perlop document to
 their equivalents in Perl 6. For full documentation on the Perl 6
-equivalents, please see the Perl 6 documentation.
+equivalents, please see the L<Perl 6 documentation|/language/operators>.
 
 =head2 Operator Precedence and Associativity
 
 The operator precedence table is somewhat different in Perl 6 than it is in
-Perl 5, so I will not detail it here. If you need to know the precedence and
+Perl 5, so it will not be detail  here. If you need to know the precedence and
```

Comment:
It could just be changed to "detailed", but I think b2gills' version is probably best.